pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -340,3 +333,7 @@ func (n *NodeLabeller) alertIfHostModelIsObsolete(originalNode *v1.Node, hostMod
 	n.recorder.Eventf(originalNode, v1.EventTypeWarning, "HostModelIsObsolete", warningMsg)
 	return nil
 }
+
+func (n *NodeLabeller) hasTSCCounter() bool {
+	return n.cpuCounter.Name == "tsc"
+}
```

Comment:
I do wonder that as well, I couldn't find a reference as to why it was decided to be implemented as such but the official libvirtxml package unmarshals it to a single object instead of a slice.

Reply:
Thank you!